KEYCLOAK-5139 refresh token does not work with pairwise subject ident…

hardselius opened the pull request:
…ifiers

hardselius committed:
KEYCLOAK-5139 refresh token does not work with pairwise subject identifiers

mposolda commented:
@hardselius Thanks for the PR! Just one minor comment, that in RefreshTokenTest.refreshTokenUserDeleted you are also testing disabling user instead of deleting him? Could you please update that one?

hardselius committed:
Update refresh token test for deleted user

Instead of disabling the user under test, the test will now correctly
delete the user under test.

mposolda commented:
@hardselius Thanks!
